achievements/563560_loc_koreana.vdf

```diff
@@ -295,24 +295,24 @@
 		"NEW_ACHIEVEMENT_1517_5_DESC"	"중형 돌격 소총으로 500의 외계 무리를 사살하십시오."
 		"NEW_ACHIEVEMENT_1517_6_NAME"	"전투 의무병"
 		"NEW_ACHIEVEMENT_1517_6_DESC"	"치료용 기관단총으로 500의 외계 무리를 사살하십시오."
-		"NEW_ACHIEVEMENT_1517_7_NAME"	"쉬운 캠페인: 남 휴머니움"
```

Comment:
you don't have to update this file, by the way. the script will automatically pull the changes from `reactivedrop_koreana.txt`

Reply:
I just know for the first time :o. thx